test_caseinsensitivedict.py

```diff
@@ -67,10 +67,18 @@ def test_copy(self):
         #     Return the value for key if key is in the dictionary, else
         #     default.  If default is not given, it defaults to None, so that
         #     this method never raises a KeyError.
+    def test_get(self):
+        myinstance = CaseInsensitiveDict()
+        myinstance['Kaisa'] = 5
+        myinstance['kaisa'] = 5
+        self.assertIn('Kaisa', myinstance)
+        self.assertNotIn('Jhin', myinstance)
+        value = myinstance.get('Kaisa')
+        nonvalue = myinstance.get('Jhin')
+        self.assertEqual(value, 5)
+        self.assertEqual(nonvalue, None)
+        self.assertEqual(myinstance['kaisa'], 5)
```

Comment:
using the `name[key]` syntax will invoke a specific method in our class, but *not* use the `get()` method. do you remember what method it will use?

the purpose of `test_get()` is to test the `get()` method, so change this here